pkg/grpc/config/config.proto

```diff
@@ -33,12 +33,18 @@ message RouteRedirect {
   optional bool strip_query = 8;
 }
 
-// Next ID: 62.
+message RouteDirectResponse {
+  uint32 status = 1;
+  string body = 2;
+}
+
+// Next ID: 63.
 message Route {
   string name = 1;
 
   string from = 2;
   repeated string to = 3;
+  RouteDirectResponse response = 62;
```

Comment:
Should we move the `RouteRedirect redirect = 34;` line into this section as well?